index.html

```diff
@@ -443,12 +443,11 @@ <h3>
           </li>
           <li>Let <var>billingAddress</var> be null.
           </li>
-          <li>If <var>request</var>.<a data-cite=
+          <li>If <var>request</var> <a data-cite=
```

Comment:
```suggestion
          <li>If <var>request</var>.<a data-cite=
```
This one is correct.